example/app/config.yml

```diff
@@ -188,6 +206,15 @@ aj:
   prefix: aj
   stackNameNoDash: AjIntegration
 
+  iams:
```

Comment:
Was adding this to 'aj' insteasd of 'cumulus-from-pr' intentional?

Reply:
Yes because Alireza's stack relies on the -integration being in there. PR does not want it in there, which was the original problem.

Reply:
👍  Understood.   That makes sense given the error in Circle.